generated/niscope/niscope/session.py

```diff
@@ -4729,6 +4779,60 @@ def export_attribute_configuration_file(self, file_path):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
+    @ivi_synchronized
+    def get_ext_cal_last_date_and_time(self):
+        '''get_ext_cal_last_date_and_time
+
+        Returns the date and time of the last external calibration performed.
+
+        Returns:
+            datetime (int): Indicates the **datetime** of the last calibration.
+
+        '''
+
+        year, month, day = self._cal_fetch_date(enums._CalibrationTypes.EXTERNAL)
+        return hightime.datetime(year, month, day)
```

Comment:
The method is named `.._date_and_time` but only the date part of the returned object is valid. Ideally, the method should be named `get_ext_cal_last_date` and it should return `date` object. To allow duck-typing with other nimi-python modules, we are using `datetime`. I guess that's a valid compromise.

The return value docstring should make it clear that only date part is valid and method docstring should say `Returns the date of the last..`.

@marcoskirsch, what do you think?

Reply:
I've added clarification in the return value docstring,